### PR TITLE
tweak: Change saved char widths

### PR DESCRIPTION
**Pull request by GMartigny**
I open this PR only to ask: Why do you keep doing the same maths operation on char widths ? I don't see the point of these two lines, but maybe I'm missing something.

### CODE REVIEW

**Comment by amio** [comment]
😄

```js
const widthTable = charWidthTable.map(w => Math.round(w * SCALE))
```
Since the width table file called "widths-verdana-11.json" so it has to be the widths of verdana in 11px size, hence here has to do the scale up work. Historically it use to do some transform work on width table, but it's not so needed for now clearly.

```js
widthTable[64] = widthTable[64] + 6
```
All char other than ascii will share this width (`@`), and emoji usually would looks a little crowd in this size, so made it a little wider for better looking for multiple emoji chars.

File: lib/widths-verdana-11.json [hunk 1]
```diff
@@ -1 +1 @@
-[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,3.9,4.3,5,9,7,12,8,3,5,5,7,9,4,5,4,5,7,7,7,7,7,7,7,7,7,7,5,5,9,9,9,6,11,7.5,7.5,7.7,8.5,7,6.3,8.5,8.3,4.6,5,7.6,6.1,9.3,8.2,8.7,6.6,8.7,7.6,7.5,6.8,8.1,7.5,11,7.5,6.8,7.5,5,5,5,9,7,7,6.6,6.9,5.7,6.9,6.6,3.9,6.9,7,3,3.8,6.5,3,11,7,6.7,6.9,6.9,4.7,5.7,4.3,7,6.5,9,6.5,6.5,5.8,7,5,7,9]
+[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,39,43,50,90,70,120,80,30,50,50,70,90,40,50,40,50,70,70,70,70,70,70,70,70,70,70,50,50,90,90,90,60,116,75,75,77,85,70,63,85,83,46,50,76,61,93,82,87,66,87,76,75,68,81,75,110,75,68,75,50,50,50,90,70,70,66,69,57,69,66,39,69,70,30,38,65,30,110,70,67,69,69,47,57,43,70,65,90,65,65,58,70,50,70,90]
```
Comment (amio) [inline lib/widths-verdana-11.json]: Need to change filename to `width-verdana-110.json` to represent this content.

**Comment by amio** [comment]
Hmm, I think we should use a separate width for none ascii chars, other than modify the `@`'s width.

**Comment by amio** [comment]
Nice! I'll check on this later, to review if `64` works fine for common cases visually.

**Comment by GMartigny** [comment]
Well indeed, it looks weird.
image

**Comment by GMartigny** [comment]
You might want to use [normalize](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/String/normalize), but I believe this belong to another change.

```js
"ì".charCodeAt() // => 236
"ì".normalize("NFD").charCodeAt() // => 105
```

I don't know the impact on perfs tho :thinking:

**Pull request merged**